src/components/SignUp/SignUpForm.js

```diff
@@ -6,14 +6,7 @@ import Select from '@material-ui/core/Select';
 import './SignUp.css';
 import Grid from '@material-ui/core/Grid';
```

Comment:
I'm thinking we can rename this component from SignUpForm to SignUpScreen just for consistency